api/src/test/java/org/jboss/shrinkwrap/api/asset/MemoryAssetTestCase.java

```diff
@@ -22,8 +22,6 @@
 import java.nio.ByteBuffer;
 import java.nio.channels.SeekableByteChannel;
 import java.nio.charset.StandardCharsets;
-import java.util.logging.Logger;
-
```

Comment:
Just a matter of taste, but I believe the extra space helps readability of certain groups of imported classes, especially from the same groupID. As such, I'd like the extra space to stay. WDYT @khermano ?